src/CommandHandlers/JoinRoomHandler.cs

```diff
@@ -15,14 +16,17 @@ public override Task Handle(Client client, NetworkObject receivedObject)
         }
         Room room = Room.GetOrAdd(roomName);
         client.SetRoom(room);
+        if (client.Room == null) return Task.CompletedTask;
 
         // set current location of user to the room id
 
         HttpClient httpClient = new();
         var locationSetRequest = new FormUrlEncodedContent(new Dictionary<string, string>
         {
             { "token", client.PlayerData.UNToken },
-            { "location", client.Room.Id.ToString() }
```

Comment:
this is not compatible with `SetBuddyLocation` endpoint in API server ... the same for all other call without `location`